container/rules/docker_toolchains.bzl

```diff
@@ -62,16 +63,43 @@ def _language_tool_layer_impl(ctx, symlinks=None, env=None, tars=None,
   files = files or ctx.files.files
   packages = packages or ctx.attr.packages
   additional_repos = additional_repos or ctx.attr.additional_repos
-  installables_tars = installables_tars or []
-  if installables_tars == [] and ctx.file.installables_tar:
-    installables_tars = [ctx.file.installables_tar]
   installation_cleanup_commands = installation_cleanup_commands or ctx.attr.installation_cleanup_commands
+  installables_tar = None
+
+  # If a link to the tarball of deb packages is passed in through
+  # --action_env=DOWNLOADABLE_TAR=<link>, use that one.
+  if "DOWNLOADABLE_TAR" in ctx.configuration.default_shell_env:
+    if "DOWNLOADABLE_TAR_SHA256" not in ctx.configuration.default_shell_env:
```

Comment:
My concern is that we will only be able to pass one single tar/sha via the flag (so if we wanted to build multiple layers in the same command this could be limiting)
Lets try to do it this way:
have the sha be defined in the BUILD rule, along with a name of the tar file, and only pass via the flag the gcs bucket?